viewer/packages/camera-manager/src/Keyboard.ts

```diff
@@ -56,6 +56,11 @@ export default class Keyboard {
     return p;
   }
 
+  public dispose(): void {
```

Comment:
Can we just move this to the bottom of the file? Would just make it a tiny bit more tidy, in my opinion :sweat_smile:

Reply:
`dispose()` is a public function, all functions below this are `private`, so I placed at the end of all `public` function :). We will be refactoring the way event handling for mouse & keyboard in near future. with [REV-454](https://cognitedata.atlassian.net/browse/REV-454)